src/vivarium/framework/results/manager.py

```diff
@@ -244,7 +239,7 @@ def register_binned_stratification(
         labels: list[str],
         excluded_categories: list[str] | None,
         target_type: str,
-        **cut_kwargs,
+        **cut_kwargs: dict[str, int | str | bool],
```

Comment:
I think you should be able to just do int | str | bool here?
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Yeah that works as well